routes/api/apiRoutes.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+
+const apiRouter = express.Router();
+
+const usersRouter = require('./users/usersRoutes.js');
+const gamesRouter = require('./games/gamesRouter.js');
+
+apiRouter.use('/users', usersRouter);
+
+apiRouter.use('/games', gamesRouter); 

```

Comment:
shouldnt we have 

apiRouter.use('/reviews', reviewsRouter);

Reply:
Oh right, we need a get all reviews endpoint. will and it by tomorrow